handbook/engineering/code-intelligence/index.md

```diff
@@ -123,6 +123,35 @@ Our team is growing and to help our new teammates have the best onboarding exper
   - [Garo Brik](../../../company/team/index.md#garo-brik-they-them)
   - [Noah Santschi-Cooney](../../../company/team/index.md#noah-santschi-cooney-he-him)
 
+## Ownership
+
+- GitHub organizations:
+
+  - [lsif](https://github.com/lsif)
+  - [sourcegraph-codeintel-showcase](https://github.com/sourcegraph-codeintel-showcase)
+
+- GitHub repositories:
+
+  - [sourcegraph/codeintelutils](https://github.com/sourcegraph/codeintelutils)
+  - [sourcegraph/code-intel-extensions](https://github.com/sourcegraph/code-intel-extensions)
+  - [sourcegraph/codeintellify](https://github.com/sourcegraph/codeintellify)
+  - [sourcegraph/lsif-go](https://github.com/sourcegraph/lsif-go)
+  - [sourcegraph/lsif-node](https://github.com/sourcegraph/lsif-node)
+  - [sourcegraph/lsif-java](https://github.com/sourcegraph/lsif-java)
+  - [sourcegraph/lsif-clang](https://github.com/sourcegraph/lsif-clang)
+  - [sourcegraph/lsif-test](https://github.com/sourcegraph/lsif-test)
+
+- BuildKite builds:
```

Comment:
```suggestion
- Buildkite builds:
```

Reply:
🙈 

Will open a PR to fix.

Reply:
https://github.com/sourcegraph/about/compare/fix-buildkite-casing?expand=1